client/platform/web-girder/views/ViewerAlert.vue

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+import { defineComponent, ref, toRef } from '@vue/composition-api';
+
+export default defineComponent({
+  name: 'ViewerAlert',
+  setup(_, { root }) {
+    const dialog = ref(false);
+
+
+    // this will change once REST updates are merged
+    const brandData = toRef(root.$store.state.Brand, 'brandData');
+
+    return {
+      dialog,
+      brandData,
+    };
+  },
+});
+</script>
+
+<template>
+  <!-- This component is for alert messages on the Viewer Page only.
+    See NavigationBar for sitewide alert banner -->
+  <v-dialog
+    v-model="dialog"
+    max-width="600px"
+  >
+    <template v-slot:activator="{ on, attrs }">
+      <v-icon
+        v-if="brandData.alertMessage"
```

Comment:
move the `v-if` up to the `v-dialog` so that when alertMessage is undefined, the whole dialog is excluded, not just its activator.